From ab0d877240008b09a93e93c0c2cfd3a91aebb72f Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Tue, 8 May 2018 17:08:56 +0800
Subject: [PATCH] PM / devfreq: rockchip_dmc: include pm_domains.h

remove rockchip_pm_register_notify_to_dmc from rockchip_dmc.h

Change-Id: I9e78eaec2a88cfc06ac3d24f01870cd39c9d05ad
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 drivers/devfreq/rockchip_dmc.c      | 1 +
 include/soc/rockchip/rockchip_dmc.h | 9 ---------
 2 files changed, 1 insertion(+), 9 deletions(-)

diff --git a/drivers/devfreq/rockchip_dmc.c b/drivers/devfreq/rockchip_dmc.c
index 092e7d626f1b..4288d44852e9 100644
--- a/drivers/devfreq/rockchip_dmc.c
+++ b/drivers/devfreq/rockchip_dmc.c
@@ -44,6 +44,7 @@
 #include <linux/suspend.h>
 #include <linux/thermal.h>
 
+#include <soc/rockchip/pm_domains.h>
 #include <soc/rockchip/rkfb_dmc.h>
 #include <soc/rockchip/rockchip_dmc.h>
 #include <soc/rockchip/rockchip_sip.h>
diff --git a/include/soc/rockchip/rockchip_dmc.h b/include/soc/rockchip/rockchip_dmc.h
index 37d381cf788f..6c1bb871c972 100644
--- a/include/soc/rockchip/rockchip_dmc.h
+++ b/include/soc/rockchip/rockchip_dmc.h
@@ -15,15 +15,6 @@
 
 #include <linux/devfreq.h>
 
-#ifdef CONFIG_ROCKCHIP_PM_DOMAINS
-int rockchip_pm_register_notify_to_dmc(struct devfreq *devfreq);
-#else
-static inline int rockchip_pm_register_notify_to_dmc(struct devfreq *devfreq)
-{
-	return 0;
-}
-#endif
-
 #ifdef CONFIG_DRM_ROCKCHIP
 int rockchip_drm_register_notifier_to_dmc(struct devfreq *devfreq);
 #else
-- 
2.35.3

